three/three.py

```diff
@@ -122,6 +130,13 @@ def italian():
     return 'tre'
 
 
+def hindi():
+    return 'teen'
+
+
+def vowel_count():
+    reutrn 2
```

Comment:
`return` typo.

Please add and verify necessary test cases to `tests/tests.py`